Makefile

```diff
@@ -164,9 +168,11 @@ k8s-install-cert-manager:
 
 .PHONY: helm-install
 helm-install: ## Install helm chart from sources.
-	helm upgrade --install --wait ytsaurus $(OPERATOR_CHART) \
+	$(HELM) upgrade --install --wait $(OPERATOR_INSTANCE) $(OPERATOR_CHART) \
+		-n $(OPERATOR_NAMESPACE) --create-namespace \
 		--set controllerManager.manager.image.repository=${OPERATOR_IMAGE} \
 		--set controllerManager.manager.image.tag=${OPERATOR_TAG}
+	$(KUBECTL) -n $(OPERATOR_NAMESPACE) rollout restart deployment -l app.kubernetes.io/instance=$(OPERATOR_INSTANCE)
```

Comment:
doesn't deployment recreate pod on helm install?

Reply:
Nope. Docker tag the same, nothing is changed in deployment - pod does not restart.